src/main/java/seedu/tp/flashcard/EventFlashcard.java

```diff
@@ -26,6 +26,7 @@ public EventFlashcard(String name, LocalDate startDate, LocalDate endDate, Strin
         assert !name.isEmpty() : "Invalid empty name!";
         assert startDate != null : "Invalid null startDate!";
         assert endDate != null : "Invalid null endDate!";
+        assert !startDate.isAfter(endDate) : "Invalid startDate and endDate";
```

Comment:
Do we have checks for this when prompting user for input? If not I think we shouldn't make the whole program fail if the user enters a start date later than the end date.

Reply:
we can implement this for v2

Reply:
Yeah, I find there is not checking when prompting user for input. And I think maybe we can deal this with exceptions in v2.0 , just as Quinn advised.

Reply:
I think I can just delete these two assertion now.